draft-ietf-scitt-architecture.md

```diff
@@ -349,12 +349,12 @@ Auditor       -->     / Collect Receipts /      /   Replay Log    /
                      '------------------'      '-----------------'
 ~~~~
 
-The SCITT architecture consists of a very loose federation of Transparency Services, and a set of common formats and protocols for issuing, registering and auditing Transparent Statements.
+The SCITT architecture consists of a very loose federation of Transparency Services, and a set of common formats and protocols for issuing, registering Signed Statements and auditing Transparent Statements.
 
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Merkle Tree Proof.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
-In this section, a high level the three main roles and associated processes in SCITT: Issuers and the Signed Statement issuance process, transparency Registry and the Transparent Statement Registration process, as well as  Verifiers and the Receipt validation process.
+In this section, a high level the three main roles and associated processes in SCITT: Issuers and the Signed Statement issuance process, transparency Registry and the Signed Statement Registration process, as well as Verifiers and the Receipt validation process.
```

Comment:
```suggestion
This section describes, at a high level, the three main roles and associated processes in SCITT: Issuers and the Signed Statement issuance process, Transparency Service and the Signed Statement Registration process, Auditors of the Signed Statements and Transparent Statements and the Validation process (including verification of signatures).
```

Reply:
Verifiers first verify the Transparent Statement, to check the inclusion in the log (offline).

Whether further Verification of Signed Statement is left to implementation, so I would remove the Signed Statement from Verifiers actions

Reply:
updated

Reply:
:+1:

Reply:
Done, please have a look!